docs/source/routing/operations/apq.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Automatic Persisted Queries (APQ)
```

Comment:
Is this content a copy-paste of what used to be in `/performance/caching/in-memory`? Trying to determine what needs a full review

Reply:
Mostly.  It brings together the "APQ" sections which previously lived separately on "Distributed caching" and "In-Memory" caching.  The intention was to change it as little as possible.